server/routes/selectPrison/prisonSelect.ts

```diff
@@ -22,10 +26,14 @@ export default abstract class PrisonSelectRoutes {
     const { user } = res.locals
     const { error } = pageData
 
-    const [prisons, prisoner] = await Promise.all([
+    const [prisons, prisoners]: [Agency[], RestrictedPatientSearchSummary[]] = await Promise.all([
       this.agencySearchService.getPrisons(user),
-      this.prisonerSearchService.getPrisonerDetails(prisonerNumber, user),
+      this.restrictedPatientSearchService.search({ searchTerm: prisonerNumber }, user),
```

Comment:
switch to using prisoner search rather than Prison API as need the supporting prison returned